packages/main/test/sap/ui/webcomponents/main/pages/Kitchen.html

```diff
@@ -19,6 +19,7 @@
 	<script src="../../../../../../resources/sap/ui/webcomponents/main/bundle.esm.js" type="module"></script>
 	<script nomodule src="../../../../../../resources/sap/ui/webcomponents/main/bundle.es5.js">
 	</script>
+	<script>delete Document.prototype.adoptedStyleSheets;</script>
```

Comment:
Do you think that this is necessary? I think we can remove it.

Reply:
We can remove it - it`s not related to the fix itself, but I did want to make another pull request just for that. At the end, I can create another PR.

Why I am adding it - the page is for testing purpose and it`s not public and I want to avoid adding it every time I want to debug the styles.